src/Faker/Calculator/Inn.php

```diff
@@ -3,7 +3,7 @@
 namespace Faker\Calculator;
 
 /**
- * @deprecated moved to ru_RU\Company, use {@link \Faker\Provider\ru_RU\Company}.
+ * @deprecated moved to ru_RU\Company, use {@link \Faker\Provider\ru_RU\Company.
```

Comment:
Interesting, not sure how to deal with this - I would probably just remove it since we have a `@see` annotation:

```suggestion
 * @deprecated
```

Reply:
You're missing the closing bracket now for `@link`.